### ISSUE TITLE
Crash on android after adding style transform

### ISSUE THREAD

**Issue by mzaien** [software-mansion/react-native-svg#2442]
### Description

I have added a style to an svg illustration to flip it on english or LRT
```tsx
                    transform: [{scaleX: I18nManager.isRTL ? 1 : -1}],
```

I solved this issue temproraly with adding different style to flip the svg

```tsx
transform: [
        { rotateY: I18nManager.isRTL ? '0deg' : '180deg' },
      ]
```

And to be honest I'm not really sure if this is a RNSVG issue or not since there are other packages that got updated to latest e.g expo , react-native , reanimated , ...etc but I had a similar environment to the one in our company's app.

here is the error log that might help
```
Your app just crashed. See the error below.
java.lang.IllegalArgumentException: width and height must be > 0
```

### Steps to reproduce

- install the repo
- do a clean prebuild
- run `yarn android`

### Snack or a link to a repository

https://github.com/mzaien/rn-expo-75-51-svg-bug

### SVG version

15.6.0

### React Native version

0.75.2

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

Expo Dev Client

### Architecture

Paper (Old Architecture)

### Build type

Debug app & dev bundle

### Device

Android emulator

### Device model

Pixel 8 

### Acknowledgements

Yes

**Comment by mzaien** [comment]
Seems it is the same issue that was fixed in https://github.com/software-mansion/react-native-svg/pull/2424 

@jakex7 can you confirm?, if so when will it be replced

**Comment by bohdanprog** [comment]
@mzaien I tried reproduce your issue, using your reproduce example.

**Comment by bohdanprog** [comment]
Hello @mzaien, I tried to reproduce your issue using your example, but it seems that I don't have the problem you described.

ScaleX: 1
![image](https://github.com/user-attachments/assets/cac74e74-1996-415e-9488-6d2e7a0d4d29)
<img src="https://github.com/user-attachments/assets/8fa7a9d4-45ac-4886-83aa-d2602a259d70" width="300px" height="700" />

ScaleX: -1
![image](https://github.com/user-attachments/assets/e62b7e65-2386-4dba-bf23-ed60689c6a3f)
<img src="https://github.com/user-attachments/assets/c8590a38-4ae7-497d-8887-2d88d8701a58" width="300px" height="700" />

**Comment by arakovskis** [comment]
This PR: https://github.com/software-mansion/react-native-svg/pull/2424 fixes crash, but applying these changes makes SVG not rendered at all when scaleX: -1 is applied. 
RN: 0.75.2 and latest svg library version.

**Comment by bohdanprog** [comment]
@arakovskis I ran your app and those environments that you provided.

**Comment by bohdanprog** [comment]
@arakovskis I ran your again your example here is the screen:
![Screenshot 2024-09-13 at 15 46 27](https://github.com/user-attachments/assets/c163eff0-6f86-42a8-a4cb-9f2f127e7514)

**Comment by marklu** [comment]
I've also encountered this issue, with a transform `scaleX: -1`. My workaround was to wrap the svg in a View with the transform for now.

**Comment by jakex7** [comment]
Hi, thanks for all the reports. I was able to reproduce the bug and have implemented a fix for this issue in #2452. It will be included in the next release.

**Comment by jakex7** [comment]
Hello,  
thank you for reporting this issue. It has been fixed in #2452 and included in the `v15.7.1` release. If for some reason the problem has not been solved, feel free to reopen the issue.

**Comment by jakex7** [comment]
It has been fixed in https://github.com/software-mansion/react-native-svg/pull/2452 and included in the v15.7.1 release. If for some reason the problem has not been solved, feel free to reopen the issue.
